modules/ocf/manifests/packages/docker/apt.pp

```diff
@@ -1,14 +1,14 @@
 # Include Docker apt repo.
 class ocf::packages::docker::apt {
   apt::key { 'docker':
-    id     => '58118E89F3A912897C070ADBF76221572C52609D',
-    server => 'keyserver.ubuntu.com',
+    id     => '9DC858229FC7DD38854AE2D88D81803C0EBFCD88',
+    source => 'https://download.docker.com/linux/debian/gpg',
   }
 
   apt::source { 'docker':
-    location    => 'http://apt.dockerproject.org/repo',
-    release     => "debian-${::lsbdistcodename}",
-    repos       => 'main',
+    location    => 'https://download.docker.com/linux/debian',
+    release     => $::lsbdistcodename,
+    repos       => 'stable',
     include   => {
       src => false
     },
```

Comment:
Hm, it doesn't hurt, but it's currently already installed in stage `first`. Would it be good style?